spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestSparkFileRewriter.java

```diff
@@ -190,9 +189,9 @@ private void checkDataFileGroupWithTooMuchData(SizeBasedDataRewriter rewriter) {
     rewriter.init(options);
 
     Iterable<List<FileScanTask>> groups = rewriter.planFileGroups(tasks);
-    Assert.assertEquals("Must have 1 group", 1, Iterables.size(groups));
+    Assertions.assertThat(Iterables.size(groups)).as("Must have 1 group").isEqualTo(1);
     List<FileScanTask> group = Iterables.getOnlyElement(groups);
-    Assert.assertEquals("Must rewrite big file", 1, group.size());
+    Assertions.assertThat(group.size()).as("Must rewrite big file").isEqualTo(1);
```

Comment:
please adjust all of these to use `Assertions.assertThat(groups).as(..).hasSize(1);`

Reply:
sure, thanks for the review. I'm working on it.